lib/service/Service.ts

```diff
@@ -144,26 +144,26 @@ class Service extends EventEmitter {
   }
 
   /**
-   * Get a list of standing peer orders from the order book for a specified trading pair, or for all
-   * trading pairs if no pair is specified.
+   * Get a list of standing orders from the order book for a specified trading pair.
    */
-  public getOrders = ({ pairId, maxResults }: { pairId?: string, maxResults?: number }) => {
-    const ret: OrderArrays = {
-      buyOrders: [],
-      sellOrders: [],
+  public getOrders = ({ pairId, maxResults }: { pairId: string, maxResults: number }) => {
+    checkArgument(pairId !== '', 'pairId must be specified');
```

Comment:
Maybe add one more `checkArgument` here to make sure maxResults is not negative.